packages/gestalt/src/Link.js

```diff
@@ -14,26 +20,83 @@ import useFocusVisible from './useFocusVisible.js';
+   */
   accessibilityLabel?: string,
+  /**
+   * Use accessibilitySelected and role when using it as a Tab. See the [Accessibility](https://gestalt.pinterest.systems/link#Accessible-Tab-Link) guidelines for more information.
+   */
   accessibilitySelected?: boolean,
+  /**
+   * Link is a wrapper around components (or children), most commonly [Text](https://gestalt.pinterest.systems/text), so that they become hiperlinks. See the [Link and Text](https://gestalt.pinterest.systems/link#Link-and-Text) variant for more information.
+   */
   children?: Node,
+  /**
+   * When `undeline` is supplied, Link's text is underlined on hover. See the [tapStyle and hoverStyle](https://gestalt.pinterest.systems/link#tapStyle-and-hoverStyle) variant for more information.
+   */
   hoverStyle?: 'none' | 'underline',
+  /**
+   * The URL that the hyperlink points to.
+   */
   href: string,
+  /**
+   * Id attribute of the anchor tag
+   */
   id?: string,
+  /**
+   * Properly positions Link relative to an inline element, such as [Text](https://gestalt.pinterest.systems/text), using the inline property.
+   */
   inline?: boolean,
+  /**
+   * Callback triggered when the user blurs Link.
+   */
   onBlur?: AbstractEventHandler<SyntheticFocusEvent<HTMLAnchorElement>>,
+  /**
+   * Callback fired when Link is clicked (pressed and released) with a mouse or keyboard. See [OnLinkNavigationProvider](https://gestalt.pinterest.systems/onlinknavigationprovider) to learn more about link navigation.
+   */
   onClick?: AbstractEventHandler<
     SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
     {| dangerouslyDisableOnNavigation: () => void |},
   >,
+  /**
+   * Callback triggered when the user focuses Link.
+   */
   onFocus?: AbstractEventHandler<SyntheticFocusEvent<HTMLAnchorElement>>,
+  /**
+   * Forward the ref to the underlying anchor element
+   */
+  ref?: Ref<'a'>,
+  /**
+   * Establishes the relationship of the linked URL. Use rel=”nofollow” for offsite links to inform search engines to ignore and not follow them
+   */
   rel?: 'none' | 'nofollow',
+  /**
+   * When supplied, Link acts a Tab. See the [Accessible Tab Link](https://gestalt.pinterest.systems/link#Accessible-Tab-Link) for more information.
+   */
   role?: 'tab',
+  /**
+   * Sets a border radius for Link. Select a rounding option that aligns with its children.
+
+Options are "circle" or "pill" for fully rounded corners or 0-8 representing the radius in boints.
```

Comment:
This sentence isn't necessary, it's just repeating the type displayed above it

Reply:
DONE